public/CNAME

```diff
@@ -0,0 +1 @@
+www.devbcn.com
```

Comment:
Why we have to remove this?

Reply:
It's the other way around, I'm adding it to this folder, to ensure it's always present on the `gh-pages` branch.

Reply:
ah, got it. But if it was not included before... why you need to add it now? I mean, the web is working right now with the CNAME in the main/master branch...

Reply:
yes, indeed, it's already on the main branch, but not on the `gh-pages` and I think that's the reason every time we publish the web the domain is reset. Adding it to the public folder will include it as part of the build files.